portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -96,6 +99,23 @@ private int getMaxCommentsToReturn(HttpServletRequest request) {
     return maxComments;
   }
 
+  /**
+   * Applies the sorting option specified by the user to the query.
+   */
+  private Query setSortOption(HttpServletRequest request, Query query) {
```

Comment:
Generally, we try not to mutate an object being passed in, but I think you gave the method an appropriate name. However, just looking at the method signature and return value, it's unclear whether you are creating a new object to return or actually modifying the object in place. If doing the latter, there isn't a need to also return the object. I would probably try to avoid this pattern altogether though and perhaps create a method like `private Query createCommentQuery(HttpServletRequest request)`.